tests/pytests/unit/utils/jinja/test_custom_extensions.py

```diff
@@ -131,10 +132,13 @@ def test_serialize_yaml():
         "baz": [1, 2, 3],
         "qux": 2.0,
         "spam": OrderedDict([("foo", OrderedDict([("bar", "baz"), ("qux", 42)]))]),
+        "tuple": ("foo", "bar"),
     }
     env = Environment(extensions=[SerializerExtension])
     rendered = env.from_string("{{ dataset|yaml }}").render(dataset=dataset)
     assert dataset == salt.utils.yaml.safe_load(rendered)
+    assert isinstance(dataset["spam"], collections.OrderedDict)
+    assert isinstance(dataset["spam"]["foo"], collections.OrderedDict)
```

Comment:
I'm going to trigger a full test run on this PR, simply to find out if there are any other tests that may fail due to these changes.